docs/guides/core-concepts/retry-ability.mdx

```diff
@@ -201,8 +198,9 @@ Cypress tries to act like a human user would using the browser.
 - Is the element behind another element?
 - Does the element have the `disabled` attribute?
 
-Actions - such as `.click()` - automatically wait until multiple built-in
-assertions like these pass, and then it will attempt to click just once.
+Action commands - such as `.click()` - automatically wait until multiple
+built-in assertions like these pass, and then it will attempt to click just
```

Comment:
```suggestion
built-in assertions like these pass, and then it will only ever be attempted once.
```
mixing example and def of action command

Reply:
Good catch. Applied as
> and then it will attempt the action once.
to keep active voice.